src/style.scss

```diff
@@ -64,12 +116,57 @@
   }
 }
 
-:host([chevron]) {
+:host([emphasis]) {
   .trigger {
-    display: flex;
+    border-left: 2px solid transparent;
+    padding-left: var(--auro-size-200);
+
+    &:focus {
+      border-left: 1px solid var(--auro-color-border-active-default) !important; // stylelint-disable-line
```

Comment:
make this only disable the specific rule that is failing so that it's clear why you are doing.